index.html

```diff
@@ -0,0 +1,590 @@
+                <li>
+                    <div class="department-content">
+                        <img src="Images/circle.png" alt="radio button" />
+                        <span>UI/UX</span>
+                    </div>
+                    <div class="count">
+                        <span>3</span> 
+                    </div>                
+                </li>
+                <li>
+                    <div class="department-content">
+                        <img src="Images/circle.png" alt="radio button" />
+                        <span>PE</span>
+                    </div>
+                    <div class="count">
+                        <span>3</span> 
+                    </div>           
+                </li>
+                <li>
+                    <div class="department-content">
+                        <img src="Images/circle.png" alt="radio button" />
+                        <span>Human Resources</span>
+                    </div>
+                    <div class="count">
+                        <span>3</span> 
+                    </div>               
+                </li>
+                <li>
+                    <div class="department-content">
+                        <img src="Images/circle.png" alt="radio button" />
+                        <span>Sales</span>
+                    </div>
+                    <div class="count">
+                        <span>3</span> 
+                    </div>               
+                </li>
+            </div>
+        </div>
+
+        <div class="section">
+            <span class="title">ROLE / USER MANAGEMENT</span>
+            <div class="all-content">
+                <div class="all-content-info">
+                    <img src="Images/lock.png" alt="lock symbol" />
+                    <span>Roles</span>
+                </div>
+                <div class="all-content-arrow">
+                    <img class="right-arrow" src="Images/right.png" alt="right pointing arrow" />
+                </div>
+            </div>
+            <div class="all-content">
+                <div class="all-content-info">
+                    <img src="Images/lock.png" alt="lock symbol" />
+                    <span>Assign User Roles</span>
+                </div>
+                <div class="all-content-arrow">
+                    <img class="right-arrow" src="Images/right.png" alt="right pointing arrow" />
+                </div>
+            </div>
+        </div>
+
+        <div class="update">
+            <div class="update-info">
+                <span>Install new update 2.3.1A</span>
+                <p>Allow to launch the new update for the AD application</p>
+            </div>
+            <div class="update-buttons">
+                <button class="dismiss">Dismiss</button>
+                <button class="accept">Accept</button>
+            </div>
+        </div>
+    </div>
+
+
+    <div class="container-body">
+        <!-- search header -->
+
+        <header>
```

Comment:
header should not be inside container body. container body should only have employees section